.gersemirc

```diff
@@ -0,0 +1,15 @@
+## Install gersemi locally:
+#  pip3 install gersemi
+## Run only on git-tracked files:
+#  gersemi -i -- $(git ls-files "*.cmake" "CMakeLists.txt" "**/CMakeLists.txt")
+
+# See "outcome configuration" section of gersemi --help for more info on these options.
+definitions: []
+disable_formatting: false
+extensions: [".gersemi/ext/builtins.py", ".gersemi/ext/cccl.py", ".gersemi/ext/thirdparty.py"]
+indent: 2
+line_length: 80
```

Comment:
Let's use the same 120 as the `.clang-format` for consistency. This way we can get more code on the screen as well.

Reply:
I just reran locally with a line limit of 120 to see the specific changes it makes and found the results significantly worse (subjectively, ofc).

Reply:
I did the same experiment now locally and I think 120 looks a LOT better. Many function calls can now again stay on a single line, which greatly improves the readability of the code, since we can fit more on a screen.

I agree this is subjectively though. Maybe we should vote on this.

Reply:
Fewer LOC at the expense of readability is not a win in my book. I'm evaluating these changes through "as a CMake developer, what results are the easiest to work with?" not "What gives us fewer lines?" Those brief stats say nothing about what effect it actually has on the changes to the content.

Per @jrhemstad's comment on the Slack thread, let's continue with the current config and revisit as more features get added upstream that will provide better control 👍